src/core/src/bootstrap/ConfigurationFactory.py

```diff
@@ -163,7 +163,7 @@ def new_prod_configuration(self, package_manager_name, package_manager_component
             },
             'package_manager': {
                 'component': package_manager_component,
-                'component_args': ['env_layer', 'composite_logger', 'telemetry_writer', 'status_handler'],
+                'component_args': ['env_layer', 'composite_logger', 'telemetry_writer', 'status_handler', 'execution_config'],
```

Comment:
Introduce 'execution_config' as the second parameter. The rest of ConfigurationFactory was once refactored to have the most commonly used components always appear at the beginning and in the same order (the only exception was if a certain component wasn't needed). Let's not drift from that for readability.